warehouse/query-core/src/main/java/datawave/query/jexl/visitors/QueryOptionsFromQueryVisitor.java

```diff
@@ -49,14 +76,58 @@ public Object visit(ASTFunctionNode node, Object data) {
     }
     
     /**
-     * If this is the f:options function, descend the tree with a List in the passed userdata. The function args will be collected into the List when visiting
-     * the child ASTStringLiteral nodes
+     * If the passed OR node contains a f:options
```

Comment:
Looks like part of this sentence may have been chopped off.

Reply:
Fixed!